test/uk/gov/hmrc/exports/base/CustomsExportsBaseSpec.scala

```diff
@@ -74,6 +70,8 @@ trait CustomsExportsBaseSpec
 
   def appConfig: AppConfig = injector.instanceOf[AppConfig]
 
+  val metrics: ExportsMetrics = injector.instanceOf[ExportsMetrics]
```

Comment:
Why this? I think using mock for tests instead of inject is better

Reply:
I want to test the metrics so have injected the original

Reply:
👍 Approved